src/pages/dapps.tsx

```diff
@@ -0,0 +1,1893 @@
+  return (
+    <Page>
+      <PageMetadata
+        title={t("common:decentralized-applications-dapps")}
+        description={t("common:page-dapps-desc")}
+        image={ogImage.src}
+      />
+      <PageHero content={heroContent} />
+      <Divider />
+      <Content>
+        <StyledH2>{t("common:get-started")}</StyledH2>
+        <Text>
+          {t("page-dapps-get-started-subtitle")}{" "}
+          <GlossaryTooltip termKey="transaction-fee">
+            {t("transaction-fees")}
+          </GlossaryTooltip>
+        </Text>
+        <Row>
+          <StepBoxContainer>
+            <StepBox to="/get-eth/">
+              <Box>
+                <StyledH3>
+                  {/* TODO: Use CSS counter for intl-friendly numbering  */}
+                  1. {t("page-wallets-get-some")}
+                </StyledH3>
+                <Text>{t("page-dapps-get-some-eth-description")}</Text>
+              </Box>
+              <ButtonSecondary
+                onClick={() =>
+                  trackCustomEvent({
+                    eventCategory: "dapp hero buttons",
+                    eventAction: "click",
+                    eventName: "get eth",
+                  })
+                }
+              >
+                {t("common:get-eth")}
+              </ButtonSecondary>
+            </StepBox>
+            <StepBox to="/wallets/find-wallet/">
+              <Box>
+                <StyledH3>2. {t("page-dapps-set-up-a-wallet-title")}</StyledH3>
+                <Text>{t("page-dapps-set-up-a-wallet-description")}</Text>
+              </Box>
+              <ButtonSecondary
+                onClick={() =>
+                  trackCustomEvent({
+                    eventCategory: "dapp hero buttons",
+                    eventAction: "click",
+                    eventName: "find wallet",
+                  })
+                }
+              >
+                {t("page-dapps-set-up-a-wallet-button")}
+              </ButtonSecondary>
+            </StepBox>
+            <StepBox to="#explore">
+              <Box>
+                <StyledH3>3. {t("page-dapps-ready-title")}</StyledH3>
+                <Text>{t("page-dapps-ready-description")}</Text>
+              </Box>
+              <ButtonPrimary
+                onClick={() =>
+                  trackCustomEvent({
+                    eventCategory: "dapp hero buttons",
+                    eventAction: "click",
+                    eventName: "go",
+                  })
+                }
+              >
+                {t("page-dapps-ready-button")}
+              </ButtonPrimary>
+            </StepBox>
+          </StepBoxContainer>
+        </Row>
+        <H3 id="beginner">
+          {t("page-dapps-beginner-friendly-header")} <Emoji text=":+1:" />
+        </H3>
+        <Text>{t("page-dapps-beginner-friendly-description")}</Text>
+        <StyledCardGrid>
+          {editorChoices.map((choice, idx) => (
+            // TODO: Override with height=80px? Component uses 100px
```

Comment:
In regards to this TODO, images being passed to `ProductCard` from `/dapps` were previously using a height of 80px (defined in graphql query)... the `ProductCard` component defaults to a size of 100px... this is used in other areas of the site, but for `/dapps` we were overriding for slightly smaller images.

Personally think we should just make this consistent at 100px (current implementation):
```suggestion
```

Reply:
cc: @nloureiro Any opposition?

Proposed:
<img width="1476" alt="image" src="https://github.com/ethereum/ethereum-org-fork/assets/54227730/da5e6c2b-8fea-4a74-b900-3b4bcab554d4">

Existing:
<img width="1468" alt="image" src="https://github.com/ethereum/ethereum-org-fork/assets/54227730/0fed0224-59ad-4dd6-b6e9-57bec4877fc5">

Personally think these look better slightly larger.

If we don't do that, we'll nee to update the `ProductCard` component to accept a sizing prop.

Reply:
All good from my end.
They do look better a bit bigger. Cool improvement.